CONTRIBUTING.md

```diff
@@ -32,3 +32,9 @@ To run the Akka Serverless TCK, see [tck/README.md].
 ## Docs
 
 Build docs in the `doc/` directory with `make`. Note that it requires a docker daemon running.
+
+## Formatting
+
+Auto-update formatting with:
+
+    prettier --config .prettierrc 'sdk/**/*.{js,ts}' 'tck/**/*.{js,ts}' 'npm-js/**/*.{js,ts}' --write
```

Comment:
Looks like there was a scripted task already set up for this:

https://github.com/lightbend/akkaserverless-javascript-sdk/blob/9553e4f904d91ceccdfa8ef98c8a4a35543b9447/package.json#L15

So could just be `npm run prettier-format` in the root directory.

Reply:
ah... maybe it's trivial then, but perhaps still good to mention auto-formatting is a thing.